package.json

```diff
@@ -63,7 +65,8 @@
     "sinon-chai": "^2.14.0",
     "stylelint": "^8.2.0",
     "stylelint-config-standard": "^17.0.0",
-    "stylelint-junit-formatter": "^0.2.1"
+    "stylelint-junit-formatter": "^0.2.1",
+    "webpack-bundle-analyzer": "^3.3.0"
```

Comment:
You shouldn't need to add the bundle analyzer here; the CLI provides it.  If you want to upgrade it, I'd bump the version in the CLI.

Reply:
Ah, okay will remove.